### PR TITLE
Resize image transformer cannot handle images with unknown pixel format.

### LINKED ISSUE #3611: Resize image transformer cannot handle images with unknown pixel format.
This results in an exception thrown by Resize transform and the entire pipeline is cancelled and user training stops. Ideally there should be a way to continue training on error, where by, we ignore the row that has an invalid data.

Alternatively, when resizing an image with unknown pixel format we can create the resized image with known pixel format such as Format32bppArgb, the same way we handled Index Pixel Format image in #3601

### PR DESCRIPTION
**Pull request by codemzs**
fixes #3611

### CODE REVIEW

**Comment by codemzs** [comment]
Image with invalid format:
taco_invalidpixelformat

Image after converting it to valid format and resizing:
z

**Comment by shauheen** [comment]
Similar to this [markdown](https://github.com/dotnet/machinelearning/blob/master/test/data/README.md), I think we should add a file in the images folder with info about the files. Can you please add something that shows where this image was obtained from and what the license is.

**Comment by codemzs** [comment]
@shauheen I have updated the license information for the new taco image that I added. It seems the readme file you pointed as an images section at the bottom.

File: src/Microsoft.ML.ImageAnalytics/ImagePixelExtractor.cs [hunk 1]
```diff
@@ -346,11 +347,15 @@ private ValueGetter<VBuffer<TValue>> GetGetterCore<TValue>(DataViewRow input, in
...
 
-                        Host.Check(src.PixelFormat == System.Drawing.Imaging.PixelFormat.Format32bppArgb
-                            || src.PixelFormat == System.Drawing.Imaging.PixelFormat.Format24bppRgb,
-                            "Transform only supports pixel formats Format24bppRgb and Format32bppArgb");
                         Host.Check(src.Height == height && src.Width == width);
 
+                        if (src.PixelFormat != PixelFormat.Format32bppArgb && src.PixelFormat != PixelFormat.Format24bppRgb)
+                        {
+                            var clone = src.Clone(new Rectangle(0, 0, src.Width, src.Height), PixelFormat.Format32bppArgb);
+                            src.Dispose();
```
Comment (eerhardt) [inline src/Microsoft.ML.ImageAnalytics/ImagePixelExtractor.cs]: If these need to be disposed, who disposes this clone?

Comment (codemzs) [reply src/Microsoft.ML.ImageAnalytics/ImagePixelExtractor.cs]: @eerhardt https://github.com/dotnet/machinelearning/blob/855a7faeb84230b01a67a72e71b4d95f8d3e3bf7/src/Microsoft.ML.ImageAnalytics/ImagePixelExtractor.cs#L328

**Pull request merged**